boards/arm/mm_feather/doc/index.rst

```diff
@@ -0,0 +1,225 @@
+| P28       | GPIO_B0_08    | D28      | GPIO2_IO08    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P29       | GPIO_B0_09    | D29      | GPIO2_IO09    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P30       | GPIO_B0_10    | D30      | GPIO2_IO10    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P31       | GPIO_B0_11    | D31      | GPIO2_IO11    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P32       | GPIO_B0_12    | D32      | GPIO2_IO12    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P33       | GPIO_B0_13    | D33      | GPIO2_IO13    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P34       | GPIO_B0_14    | D34      | GPIO2_IO14    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P35       | GPIO_B0_15    | D35      | GPIO2_IO15    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+|           | GPIO_AD_B1_07 |          | GPIO1_IO23    |          |               |
++-----------+---------------+----------+---------------+ I2C0     | I2C_3         |
+|           | GPIO_AD_B1_06 |          | GPIO1_IO22    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+|           | GPIO_AD_B1_00 |          | GPIO1_IO16    |          |               |
++-----------+---------------+----------+---------------+ I2C1     | I2C_1         |
+|           | GPIO_AD_B1_00 |          | GPIO1_IO17    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+
+
+Programming and Flash
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Console
+=====================
+
+Connect a USB-to-serial adapter from your PC to corresponding UART pins of SwiftIO Feather.
+
+Use the following settings with your serial terminal of choice (minicom, putty,
+etc.):
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Flashing
+========
+
+Here is an example for the :ref:`hello_world` application.
+
+Connect a debuger from your PC to corresponding SWD pins of SwiftIO Feather.
+On Ubuntu, DAPLink debug probes appear on the host
+computer as a USB disk mounted to ``/media/<user>/DAPLink/``,
+where ``<user>`` is your login name.
+
+.. code-block:: console
+
+    west build -b mm_feather samples/hello_world
+    cp build/zephyr/zephyr.bin /media/<user>/DAPLink/
```

Comment:
Have you tried enabling the jlink or pyocd runners to use `west debug` and `west flash`?

Reply:
The early pyocd had a bug in supporting rt1052 QSPI flash, which was resolved in September last year https://github.com/pyocd/pyOCD/pull/845. Therefore, pyocd can support west flash and west debug. Thank you for your reminder. I will update the document